pyintelowl/pyintelowl.py

```diff
@@ -184,6 +184,31 @@ def get_analyzer_configs(self):
             errors.append(str(e))
         return {"errors": errors, "answer": answer}
 
+    def get_all_tags(self):
+        errors = []
+        try:
+            url = self.instance + "/api/tags"
+            response = self.session.get(url)
+            logger.debug(response.url)
+            response.raise_for_status()
+            answer = response.json()
+        except Exception as e:
+            errors.append(str(e))
+        return errors, answer
```

Comment:
please return in the format `answers, errors`. Same for the other function.

Reply:
ah, sure! will change this

Reply:
I am sorry maybe I was not clear, I had just meant to reorder the tuple (the `answer` should be on the LHS) not change it into dictionary format.